english/cry/cry.md

```diff
@@ -0,0 +1,3 @@
+As you cry and cry, your tears fill the cup you're holding and soon you have a cup full of salty tears. And you can't do much with that!
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.